ui/marketplace/MarketplaceAppTopBar.tsx

```diff
@@ -85,6 +70,25 @@ const MarketplaceAppTopBar = ({ data, isLoading, isWalletConnected, securityRepo
             { getHostname(data?.url) }
           </chakra.span>
         </LinkExternal>
+        <Skeleton isLoaded={ !isLoading }>
+          <MarketplaceAppInfo data={ data }/>
+        </Skeleton>
+        { (isExperiment && (securityReport || isLoading)) && (
```

Comment:
```suggestion
        { (isExperiment && securityReport) && (
```

I assume it should be enough to check only the presence of the `securtiyReport` data.

Reply:
`securityReport` is `undefined` while the loading is in progress, so in this case everything is correct here 🙂